thrift/src/main/java/io/grpc/thrift/MessageFactory.java

```diff
@@ -31,11 +31,14 @@
 
 package io.grpc.thrift;
 
+import io.grpc.ExperimentalApi;
+
 import org.apache.thrift.TBase;
 
 /** 
  *  Produce new Message Instances. Used by Marshaller to deserialize incoming messages. 
  */
+@ExperimentalApi("https://github.com/grpc/grpc-java/issues/2170")
```

Comment:
Sorry, I was misreading the other file and I thought there was an inconsistency. No, I was wrong. Please ignore my comment.